library/pk_wrap.c

```diff
@@ -737,6 +737,106 @@ static int ecdsa_verify_wrap( void *ctx, mbedtls_md_type_t md_alg,
 }
 #endif /* MBEDTLS_USE_PSA_CRYPTO */
 
+#if defined(MBEDTLS_USE_PSA_CRYPTO)
+static int pk_ecdsa_sig_asn1_from_psa( unsigned char *sig, size_t *sig_len,
+                                       size_t buf_len );
+
+static int find_ecdsa_private_key( unsigned char **buf, unsigned char *end, size_t *key_len)
```

Comment:
Can you add some documentation for this function? Something similar to what's done for `pk_ecdsa_sig_asn1_from_psa()`.